### PR TITLE
news 217 : add French translation

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Review by bitschmidty — COMMENTED** [review]
I pushed a commit which fixes 1. trailing spaces errors and 2. extra quote " in the PR review section. I also rebased on master. Let me know if all looks good to you @Copinmalin !

**Comment by Copinmalin** [comment]
Done all is OK for me

**Comment by Copinmalin** [comment]
Does someone can check if all is ok to publish please ?

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  des en-têtes) en ajoutant tous les pairs qui nous annoncent un bloc via un `inv`
+  comme pairs de synchronisation des en-têtes ?"
+  a1="Chacun de ces pairs commencera alors à nous envoyer le même flux d'en-têtes:
+  le `inv` déclenche un `getheaders` vers le même pair, et sa réponse `headers` déclenche
+  un autre `getheaders` pour la plage d'en-têtes de bloc immédiatement suivante. Recevoir
+  des en-têtes en double est inoffensif, sauf pour la bande passante supplémentaire."
+  a1link="https://bitcoincore.reviews/25720#l-62"
+
+  q2="Quelle serait votre estimation (limite inférieure/supérieure) de la quantité de bande
+  passante est gaspillée ?"
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:64]: s/est/qui est/ ou omet "est"
```suggestion
  passante gaspillée ?"
```

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  en-têtes ne prend que quelques minutes)"
+  a2link="https://bitcoincore.reviews/25720#l-79"
+
+  q3="Quel est la proposition des membres du CNodeState avec `fSyncStarted` et `m_headers_sync_timeout`,
+  et `PeerManagerImpl::nSyncStarted` ?
+  Si nous commençons à synchroniser les en-têtes avec les pairs qui nous annoncent un bloc
+  via un `inv`, pourquoi ne pas augmenter `nSyncStarted` et mettre `fSyncStarted = true`
+  et mettre à jour `m_headers_sync_timeout` ?"
+  a3="`nSyncStarted` compte le nombre de pairs dont `fSyncStarted` est vrai, et ce nombre
+  ne peut pas être supérieur à 1 jusqu'à ce que le noeud ait des en-têtes proches (dans un délai de jour)
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:77]: d'un jour (ou) de jours, didn't check which
```suggestion
  ne peut pas être supérieur à 1 jusqu'à ce que le noeud ait des en-têtes proches (dans un délai d'un jour)
```

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  de l'heure actuelle. Ce pair (arbitraire) sera notre pair initial de synchronisation d'en-têtes.
+  Si ce pair est lent, le noeud l'arrête (`m_headers_sync_timeout`) et trouve un autre pair de
+  synchronisation d'en-têtes 'initial'. Mais si, pendant la synchronisation des en-têtes, un noeud
+  envoie un message `inv` qui annonce des blocs, alors sans ce PR, le noeud commencera à demander des
+  en-têtes à ce pair aussi, _sans_ mettre son drapeau `fSyncStarted`. Ceci est la source des messages
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:82]: "flag" ici veut ne veut pas dire drapeau mais le sens informatique
```suggestion
  en-têtes à ce pair aussi, _sans_ mettre son flag `fSyncStarted`. Ceci est la source des messages
```

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  d'être réactifs. Un autre avantage est qu'un pair qui réussit à nous envoyer le bloc `inv`
+  en premier est souvent aussi un pair très rapide. Nous ne choisirons donc pas un autre pair
+  lent si, pour une raison quelconque, notre pair initial est lent."
+
+  q4link="https://bitcoincore.reviews/25720#l-135"
+%}
+
+## Mise à jour et candidat à la publication
+
+*Nouvelles mises à jour et candidat à la publication pour le projet populaire d'infrasture
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:102]: - "populaire" isn't a good translation of "popular"

- infrastructure

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+de Bitcoin. S'il vous plait prévoyez de mettre à jour à la nouvelle mise à jour ou d'aider
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:103]: "mettre à jour à la nouvelle mise à jour" is redundant

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+sans clés privées. Avant d'appeler cette fonction, lisez la [documentation][managing wallets]
+et sachez qu'il existe certaines différences d'API entre les portefeuilles sans descripteurs
+et ceux qui prennent en charge nativement les descripteurs.
+
+<!-- TODO:harding to separate dual funding from interactive funding -->
+
+- [Eclair #2406][] ajoute une option pour configurer l'implémentation expérimentale
+  [interactive funding protocol][topic dual funding] pour exiger que les transactions
+  d'ouverture de canal incluent uniquement les *entrées confirmées* --- entrées qui
+  dépensent les sorties font partie d'une transaction confirmée. S'il est activé,
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:148]: je suis pas certain mais ca semble manquer un mot ici
```suggestion
  dépensent les sorties qui font partie d'une transaction confirmée. S'il est activé,
```

**Review by jonatack — COMMENTED** [review]
Quick readover ;)

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 6 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: diverses adaptations faites, merci :-)

**Comment by bitschmidty** [comment]
Ill defer to @jonatack on any final translation feedback before merging.

**Comment by Copinmalin** [comment]
@jonatack hello, I made the changes, can you look at it and tell me if it's OK please ?

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+title: 'Bitcoin Optech Newsletter #217'
+permalink: /fr/newsletters/2022/09/14/
+name: 2022-09-14-newsletter-fr
+slug: 2022-09-14-newsletter-fr
+type: newsletters-fr
+layout: newsletters-fr
+lang: fr
+---
+La newsletter de cette semaine comprend notre section habituelle avec le
+résumé d'une réunion du Bitcoin Core PR Review Club une liste de nouvelles mises à jour des logiciels et des candidats
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:11]:
```suggestion
résumé d'une réunion du Bitcoin Core PR Review Club, une liste de nouvelles mises à jour des logiciels et des candidats
```

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+à la publication, et des résumés de changements notables apportés aux principaux projets
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:12]: A propos les 2 expressions "candidats a la publication" : je suis pas certain que "publication" soit l'equivalent juste de "release", plutot :

- une release candidate
- une version candidate

Voir plus : 
- https://tr-ex.me/traduction/anglais-fran%C3%A7ais/release+candidate
- https://fr.glosbe.com/en/fr/release%20candidate

**Review by jonatack — COMMENTED** [review]
WIP re-reviewing

**Comment by jonatack** [comment]
Suggestions.
```diff
@@ -27,25 +27,25 @@ de la réunion.*
 [Réduire la bande passante pendant la synchronisation initiale des en-têtes
 lorsqu'un bloc est trouvé][review club 25720] est une proposition d'amélioration (PR)
 de Suhas Daftuar qui réduit la demande de bande passante d'un nœud lors de la synchronisation
-de la blockchain avec les pairs, y compris pendant le téléchargement du bloc initial (IBD).
+de la blockchain avec les pairs, y compris pendant le téléchargement de blocs initial (IBD).
 Une partie importante de l'éthique du Bitcoin est de minimiser les demandes de ressources
 pour faire fonctionner un nœud complet de validation, y compris les ressources réseau,
 afin d'encourager plus d'utilisateurs à faire fonctionner des nœuds complets. Accélérer le temps de
 synchronisation favorise également cet objectif.
 
-La synchronisation de la blockchain se fait en deux phases: Premièrement, le nœud
-reçoit des en-têtes de bloc de la part de ses pairs; ces en-têtes sont suffisantes
-pour déterminer la meilleure chaîne (probable) (celle qui a le plus de travail).
+La synchronisation de la blockchain se fait en deux phases : premièrement, le nœud
+reçoit des en-têtes de blocs de la part de ses pairs ; ces en-têtes sont suffisantes
+pour déterminer la meilleure chaîne probable, celle qui cumule le plus de travail (proof of work).
 Deuxièmement, le nœud utilise cette meilleure chaîne d'en-têtes pour demander et
 télécharger les blocs complets correspondants.
-Cette demande d'amélioration (PR) n'affecte que la première phase (téléchargement
+Cette proposition d'amélioration (PR) n'affecte que la première phase (le téléchargement
 des en-têtes).
 
 {% include functions/details-list.md
   q0="Pourquoi les nœuds reçoivent-ils (la plupart du temps) des annonces de bloc `inv`
   pendant qu'ils effectuent la synchronisation initiale des en-têtes, même s'ils ont indiqués
   leur préférence pour les annonces d'en-têtes ([BIP 130][])?"
-  a0="Un nœud n'annoncera pas un nouveau bloc à un pair en utilisant un message d'en-tête
+  a0="Un nœud n'annoncera pas de nouveau bloc à un pair en utilisant un message d'en-tête
   si le pair n'a pas précédemment envoyé un en-tête auquel le nouvel en-tête est connecté,
   et les nœuds de synchronisation n'envoient pas d'en-têtes."
   a0link="https://bitcoincore.reviews/25720#l-30"
@@ -54,26 +54,26 @@ des en-têtes).
   des en-têtes) en ajoutant tous les pairs qui nous annoncent un bloc via un `inv`
   comme pairs de synchronisation des en-têtes ?"
   a1="Chacun de ces pairs commencera alors à nous envoyer le même flux d'en-têtes:
-  le `inv` déclenche un `getheaders` vers le même pair, et sa réponse `headers` déclenche
+  les messages `inv` déclenchent un `getheaders` vers le même pair, et sa réponse `headers` déclenche
   un autre `getheaders` pour la plage d'en-têtes de bloc immédiatement suivante. Recevoir
   des en-têtes en double est inoffensif, sauf pour la bande passante supplémentaire."
   a1link="https://bitcoincore.reviews/25720#l-62"
 
-  q2="Quelle serait votre estimation (limite inférieure/supérieure) de la quantité de bande
-  passante est gaspillée ?"
-  a2="la limite supérieure (en bytes): `(number_peers - 1) * number_blocks * 81`;
+  q2="Quelle serait votre estimation (limite inférieure / supérieure) de la quantité de bande
+  passante gaspillée ?"
+  a2="La limite supérieure (en bytes): `(quantité de pairs - 1) * quantité de blocs * 81` ;
   la limite inférieure: zero (si aucun nouveau bloc n'arrive pendant la synchronisation des
   en-têtes ; si le pair de synchronisation et le réseau sont rapides, le téléchargement des 700k+
-  en-têtes ne prend que quelques minutes)"
+  en-têtes ne prendra que quelques minutes)."
   a2link="https://bitcoincore.reviews/25720#l-79"
 
-  q3="Quel est la proposition des membres du CNodeState avec `fSyncStarted` et `m_headers_sync_timeout`,
-  et `PeerManagerImpl::nSyncStarted` ?
+  q3="A quoi servent les membres de CNodeState `fSyncStarted` et `m_headers_sync_timeout` ainsi que
+  `PeerManagerImpl::nSyncStarted` ?
   Si nous commençons à synchroniser les en-têtes avec les pairs qui nous annoncent un bloc
   via un `inv`, pourquoi ne pas augmenter `nSyncStarted` et mettre `fSyncStarted = true`
   et mettre à jour `m_headers_sync_timeout` ?"
   a3="`nSyncStarted` compte le nombre de pairs dont `fSyncStarted` est vrai, et ce nombre
-  ne peut pas être supérieur à 1 jusqu'à ce que le noeud ait des en-têtes proches (dans un délai d'un jour)
+  ne peut pas être supérieur à 1 jusqu'à ce que le noeud ait des en-têtes proches (moins d'un jour)
   de l'heure actuelle. Ce pair (arbitraire) sera notre pair initial de synchronisation d'en-têtes.
   Si ce pair est lent, le noeud l'arrête (`m_headers_sync_timeout`) et trouve un autre pair de
   synchronisation d'en-têtes 'initial'. Mais si, pendant la synchronisation des en-têtes, un noeud
@@ -90,7 +90,7 @@ des en-têtes).
   de l'approche adoptée dans le PR par rapport à cette alternative ?"
   a4="Un avantage est que les pairs qui nous annoncent un `inv` ont une plus grande probabilité
   d'être réactifs. Un autre avantage est qu'un pair qui réussit à nous envoyer le bloc `inv`
-  en premier est souvent aussi un pair très rapide. Nous ne choisirons donc pas un autre pair
+  en premier est souvent aussi un pair très rapide. Nous ne choisirons donc pas d'autre pair
   lent si, pour une raison quelconque, notre pair initial est lent."
 
   q4link="https://bitcoincore.reviews/25720#l-135"
@@ -98,13 +98,13 @@ des en-têtes).
 
 ## Mise à jour et candidat à la publication
 
-*Nouvelles mises à jour et candidat à la publication pour le projet principal d'infrastructure
-de Bitcoin. S'il vous plait prévoyez de mettre à jour à la nouvelle version ou d'aider
-à tester le candidat à publication.*
+*Nouvelles mises à jour et release-candidates des projets principaux d'infrastructure
+Bitcoin. Prévoyez s'il vous plait de vous mettre à jour à la nouvelle version ou d'aider
+à tester les release-candidates.*
 
-- [LDK 0.0.111][] ajoute un support pour la création, la réception et la relayage
+- [LDK 0.0.111][] ajoute la création, la réception et la relayage d'
   [onion messages][topic onion messages], parmi plusieurs autres nouvelles fonctionnalités
-  et de corrections de bogues.
+  et apporte des corrections de bogues.
 
 ## Changements notables dans le code et la documentation
 
@@ -121,12 +121,12 @@ Proposals (BIPs)][bips repo], et [Lightning BOLTs][bolts repo].*
   timedata, et torcontrol.
 
 - [Bitcoin Core #25768][] corrige un bogue où le portefeuille ne rediffusait
-pas toujours les transactions enfants des transactions non confirmées.
-Le portefeuille intégré de Bitcoin Core tente périodiquement de diffuser l'une
-de ses transactions qui n'a pas encore été confirmée. Certaines de ces
+pas toujours de transactions enfants des transactions non confirmées.
+Le portefeuille intégré de Bitcoin Core tente périodiquement de diffuser ses
+transactions qui n'ont pas encore été confirmées. Certaines de ces
 transactions peuvent dépenser les sorties d'autres transactions non confirmées.
-Bitcoin Core mélangeait l'ordre des transactions avant de les envoyer à un autre
-sous-système Bitcoin Core qui s'attendait à recevoir des transactions parentes
+Bitcoin Core rendait aléatoire l'ordre des transactions avant de les envoyer à une autre
+partie du code qui s'attendait à recevoir des transactions parentes
 non confirmées avant les transactions enfants (ou, plus généralement, tous les
 ancêtres non confirmés avant tout descendant). Lorsqu'une transaction enfant
 était reçue avant son parent, elle était rejetée en interne au lieu d'être rediffusée.
@@ -135,7 +135,7 @@ ancêtres non confirmés avant tout descendant). Lorsqu'une transaction enfant
 en utilisant nativement [descriptors][topic descriptors]. Cela fonctionne pour
 les portefeuilles pré-HD (ceux créés avant que [BIP32][] n'existe ou n'ait été adopté
 par Bitcoin Core), les portefeuilles HD et les portefeuilles de surveillance uniquement
-sans clés privées. Avant d'appeler cette fonction, lisez la [documentation][managing wallets]
+sans clés privées (watch-only). Avant d'utiliser cette RPC, lisez la [documentation][managing wallets]
 et sachez qu'il existe certaines différences d'API entre les portefeuilles sans descripteurs
 et ceux qui prennent en charge nativement les descripteurs.
 
@@ -146,7 +146,7 @@ et ceux qui prennent en charge nativement les descripteurs.
   d'ouverture de canal incluent uniquement les *entrées confirmées* --- entrées qui
   dépensent les sorties faisant partie d'une transaction confirmée. S'il est activé,
   cela peut empêcher un initiateur de retarder l'ouverture d'un canal en le basant
-  sur une grande transaction non confirmée avec un faible taux de commission.
+  sur une grande transaction non confirmée avec un faible taux de commission (fee rate).
```

It would also be good to settle on either the word "block" or "bloc" consistently. Since the word "blockchain" is frequently used, I'd go for "block" over "bloc" but no strong opinion.

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+  en-têtes ne prend que quelques minutes)"
+  a2link="https://bitcoincore.reviews/25720#l-79"
+
+  q3="Quel est la proposition des membres du CNodeState avec `fSyncStarted` et `m_headers_sync_timeout`,
+  et `PeerManagerImpl::nSyncStarted` ?
+  Si nous commençons à synchroniser les en-têtes avec les pairs qui nous annoncent un bloc
+  via un `inv`, pourquoi ne pas augmenter `nSyncStarted` et mettre `fSyncStarted = true`
+  et mettre à jour `m_headers_sync_timeout` ?"
+  a3="`nSyncStarted` compte le nombre de pairs dont `fSyncStarted` est vrai, et ce nombre
+  ne peut pas être supérieur à 1 jusqu'à ce que le noeud ait des en-têtes proches (dans un délai d'un jour)
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:76]: je crois qu "un jour" n'est pas le delai mais l'ecart de l'heure actuelle pour preciser ce que veut dire "proche" ici

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+
+  q2="Quelle serait votre estimation (limite inférieure/supérieure) de la quantité de bande
+  passante est gaspillée ?"
+  a2="la limite supérieure (en bytes): `(number_peers - 1) * number_blocks * 81`;
+  la limite inférieure: zero (si aucun nouveau bloc n'arrive pendant la synchronisation des
+  en-têtes ; si le pair de synchronisation et le réseau sont rapides, le téléchargement des 700k+
+  en-têtes ne prend que quelques minutes)"
+  a2link="https://bitcoincore.reviews/25720#l-79"
+
+  q3="Quel est la proposition des membres du CNodeState avec `fSyncStarted` et `m_headers_sync_timeout`,
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:70]: s/proposition/utilité/ 

A quoi servent les membres ...

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+
+- [Bitcoin Core #25614][] construit sur [Bitcoin Core #24464][], permettant
+  la possibilité d'ajouter et de tracer des journaux avec des niveaux spécifiques
+  de gravité dans addrdb, addrman, banman, i2p, mempool, netbase, net, net_processing,
+  timedata, et torcontrol.
+
+- [Bitcoin Core #25768][] corrige un bogue où le portefeuille ne rediffusait
+pas toujours les transactions enfants des transactions non confirmées.
+Le portefeuille intégré de Bitcoin Core tente périodiquement de diffuser l'une
+de ses transactions qui n'a pas encore été confirmée. Certaines de ces
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:126]: I believe it's not one but any (i.e. all) of its unconfirmed transactions.

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 8 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-14-newsletter.md:12]: une pré-version pourrait convenir ?

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 11 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: Hello, I did all modification, if you can check one more time,
Sorry for this supplementary job,

**Comment by Copinmalin** [comment]
Hello @jonatack, could you review my last proposition please ?

File: _posts/fr/newsletters/2022-09-14-newsletter.md [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+Proposals (BIPs)][bips repo], et [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25614][] construit sur [Bitcoin Core #24464][], permettant
+  la possibilité d'ajouter et de tracer des journaux avec des niveaux spécifiques
+  de gravité dans addrdb, addrman, banman, i2p, mempool, netbase, net, net_processing,
+  timedata, et torcontrol.
+
+- [Bitcoin Core #25768][] corrige un bogue où le portefeuille ne rediffusait
+pas toujours de transactions enfants des transactions non confirmées.
+Le portefeuille intégré de Bitcoin Core tente périodiquement de diffuser
```
Comment (jonatack) [inline _posts/fr/newsletters/2022-09-14-newsletter.md:124]:
```suggestion
Le portefeuille intégré de Bitcoin Core tente périodiquement de diffuser
```

**Review by jonatack — COMMENTED** [review]
Changes look good to me, including the additional fixups you saw and improved as well!

One trailing white space <strike>and need to remove the non-pertinent changes from the git history</strike> (edit, I may have been confused by all the commits).

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 8 above)
Comment (jonatack) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: > une pré-version pourrait convenir ?

Je ne sais pas :) entre "pré-version" et "release-candidate" ou "version candidate" je ne connais que les derniers -- les devs utilisaient pas mal de franglais mais ca fait quelques annees que je ne travaille plus en francais alors je ne suis plus tres au courant des pratiques.

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 12 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: Done

Comment (bitschmidty) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: Thanks @jonatack and @Copinmalin just was reviewing and saw the trailing space as well, thanks for correcting.

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 8 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: > 

Comment est-il possible de savoir ? Je suis parti sur "pré-version" qui me parait compréhensible et français

Comment (jonatack) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: Peut-etre demander a des collegues devs francais ou faire un sondage sur twitter.

Comment (jonatack) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: I don't think it's a blocker. The current version could be merged and you could make a later PR could update these terms if needed.

**Review by bitschmidty — COMMENTED** [review]
In the front matter, these should be the field values (even with fr):

type: newsletter
layout: newsletter

Pushed a change for this. It didn’t affect the build, but the rendering didn’t grab the correct layout/template.

File: _posts/fr/newsletters/2022-09-14-newsletter.md (hunk 8 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: je fais un sondage Twitter quand même pour les prochaines fois :-)
https://twitter.com/copinmalin/status/1574376615810932738?s=20&t=mnb7Cr-9VP2K250DqOajJQ

Comment (jonatack) [reply _posts/fr/newsletters/2022-09-14-newsletter.md]: > je fais un sondage Twitter quand même pour les prochaines fois :-) https://twitter.com/copinmalin/status/1574376615810932738?s=20&t=mnb7Cr-9VP2K250DqOajJQ

Cool, retweeted and voted.

**Comment by bitschmidty** [comment]
Thanks for the translation, @Copinmalin !

**Pull request merged**